widgetssdk/src/main/java/com/glia/widgets/call/CallView.kt

```diff
@@ -809,12 +807,13 @@ internal class CallView(
     }
 
     private fun showOverlayPermissionsDialog() {
```

Comment:
And shouldn't these dialog type definitions be somewhere else entirely? Why are they defined in View classes. This way they are not reusable. Maybe define them in Dialogs.kt instead and the view only adds listeners?